cmd/bbolt/command_buckets.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	bolt "go.etcd.io/bbolt"
+)
+
+// newBucketsCommand creates a new command that prints a list of buckets in the given Bolt database.
+//
+// The path to a Bolt database must be specified as an argument.
+func newBucketsCommand() *cobra.Command {
+	var bucketsCmd = &cobra.Command{
+		Use:   "buckets <path>",
+		Short: "Print a list of buckets",
+		Long:  "Print a list of buckets in the given Bolt database\nThe path to a Bolt database must be specified as an argument",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return printBucketsList(cmd, args[0])
+		},
+	}
+	return bucketsCmd
+}
+
+// printBucketsList prints a list of buckets in the given Bolt database.
+func printBucketsList(cmd *cobra.Command, path string) error {
+	// Required database path.
+	if path == "" {
+		return ErrPathRequired
+		// Verify if the specified database file exists.
+	} else if _, err := os.Stat(path); os.IsNotExist(err) {
+		return ErrFileNotFound
+	}
```

Comment:
Please use `checkSourceDBPath`. See example below,
https://github.com/etcd-io/bbolt/blob/cb0618b9da2b436b7bed6013b515697cbaeb458a/cmd/bbolt/command_inspect.go#L27

Reply:
I used the original code but I can look at changing it to the above